lib/java/polyglot-ydoc-server/src/test/java/org/enso/polyfill/EncodingPolyfillTest.java

```diff
@@ -0,0 +1,78 @@
+package org.enso.polyfill;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import org.enso.polyfill.encoding.EncodingPolyfill;
+import org.graalvm.polyglot.Context;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class EncodingPolyfillTest {
+
+    private Context context;
+    private ExecutorService executor;
+
+    public EncodingPolyfillTest() {
+    }
+
+    @Before
+    public void setup() throws Exception {
+        executor = Executors.newSingleThreadExecutor();
+        var encoding = new EncodingPolyfill();
+        var b = Context.newBuilder("js");
+
+        var chromePort = Integer.getInteger("inspectPort", -1);
+        if (chromePort > 0) {
+            b.option("inspect", ":" + chromePort);
```

Comment:
Out of curiosity. How does one pass the `inspectPort` to the test via an `sbt` command?

Reply:
```
sbt> eval System.setProperty("inspectPort", "666")
```
should work